src/interpreter.js

```diff
@@ -0,0 +1,61 @@
+const bscript = require('bscript-parser')
+
+const Interpreter = module.exports = class Interpreter {
+  constructor (script) {
+    this.script = script
+    this.pc = 0
```

Comment:
Is it possible to use more descriptive attributes than `pc` and `pend`?  I have no idea what they mean.

Reply:
Yeah, sorry those are tenebra from Bitcoin core. I think it means program counter and program end.

Reply:
Got it.  Definitely needs to be a more descriptive property or a have a comment to explain what they stand for.